packages/x-components/src/design-system/components/suggestion-group/default.scss

```diff
@@ -8,7 +8,6 @@
   box-sizing: border-box;
```

Comment:
1. Could we add an example of this to the `design-system-suggestion-group.vue`?
2. Should we fix the long text in the tag?

![image](https://user-images.githubusercontent.com/68222542/149499745-6a3ddb1b-2b14-45a3-854d-d04079cce786.png)

```html
    <article class="x-list x-list--wrap x-list--gap-06 x-list--align-start">
      <h2 class="x-title2">Default + Tag</h2>

      <div class="x-suggestion-group x-tag">
        <div class="x-suggestion">
          <span class="x-suggestion__query">Suggestion</span>
        </div>
        <button class="x-button">
          <CrossIcon />
        </button>
      </div>

      <div class="x-suggestion-group x-tag">
        <div class="x-suggestion x-suggestion--matching">
          <span class="x-suggestion__query">
            Suggestion
            <span class="x-suggestion__matching-part">Query</span>
          </span>
        </div>
        <button class="x-button">
          <CrossIcon />
        </button>
      </div>

      <div class="x-suggestion-group x-tag">
        <button class="x-button">
          <CrossIcon />
        </button>
        <div class="x-suggestion x-suggestion--matching">
          <span class="x-suggestion__query">
            Suggestion
            <span class="x-suggestion__matching-part">Query</span>
          </span>
        </div>
      </div>

      <div class="x-suggestion-group x-tag">
        <div class="x-suggestion x-suggestion--matching">
          <SearchIcon />
          <span class="x-suggestion__query">
            Suggestion
            <span class="x-suggestion__matching-part">Query</span>
          </span>
        </div>
        <button class="x-button">
          <CrossIcon />
        </button>
      </div>

      <div class="x-suggestion-group x-tag">
        <button class="x-button">
          <CrossIcon />
        </button>
        <div class="x-suggestion x-suggestion--matching">
          <span class="x-suggestion__query">
            Suggestion
            <span class="x-suggestion__matching-part">Query</span>
          </span>
          <SearchIcon />
        </div>
      </div>

      <div class="x-suggestion-group x-tag">
        <button class="x-button">
          <CrossIcon />
        </button>
        <div class="x-suggestion x-suggestion--matching">
          <span class="x-suggestion__query">
            Suggestion extra large name
            <span class="x-suggestion__matching-part">Query</span>
          </span>
          <SearchIcon />
        </div>
      </div>
    </article>
```

Reply:
I added the example, but I'm not sure about the second one, maybe we can add a utility to do ellipsis in the text?

Reply:
the long text wrapping is intended. That's whats must happen when the text is too long in a tag.
And we already have a utility `x-ellipsis` for the ellipsis in `src/design-system/components/typography/default.scss.`
Maybe we have to add an example for that on the typography section.